deploy/quickstart-k8s.yaml

```diff
@@ -38,6 +38,9 @@ rules:
   - apiGroups: ["everest.percona.com"]
     resources: ["*"]
     verbs: ["*"]
+  - apiGroups: ["operators.coreos.com"]
+    resources: ["installplans"]
+    verbs: ["*"]
```

Comment:
Can we be less broad and just have the verbs we actually need?

Reply:
Done